public/docs/_examples/template-syntax/dart/lib/app_component.dart

```diff
@@ -0,0 +1,122 @@
+library template_syntax.app_component;
+
+import 'dart:html';
+import 'dart:convert';
+
+import 'package:angular2/angular2.dart';
+import 'package:template_syntax/hero.dart';
+import 'package:template_syntax/hero_detail_component.dart';
+import 'package:template_syntax/click_me_component.dart';
+import 'package:template_syntax/loop_back_component.dart';
+import 'package:template_syntax/key_up_components.dart';
+import 'package:template_syntax/little_hero.dart';
+
+List<Hero> _MockHeroes = [
+  new Hero('Hercules',
+      lastName: 'Son of Zeus',
+      birthdate: new DateTime(1970, 1, 25),
+      url: 'http://www.imdb.com/title/tt0065832/',
+      rate: 325),
+  new Hero('eenie', lastName: 'toe'),
+  new Hero('Meanie', lastName: 'Toe'),
+  new Hero('Miny', lastName: 'Toe'),
+  new Hero('Moe', lastName: 'Toe')
+];
+
+enum _Color { Red, Green, Blue }
```

Comment:
@kwalrath .Yep, at the very bottom of `index.html` in the TS example app there is a blurb about using enums in bindings. Thats the part that references this enum and the `colorToggle()` method. What I meant to say was I don't see any reference to it in the actual [tutorial](https://angular.io/docs/ts/latest/guide/template-syntax.html). So I'm not sure if it was just scrubbed, or if it will be referenced at a later time.
On this note, I had a heck of a time trying to figure out how to use enums in Dart in the way that they are in TS. I'm not sure that what I ended up with is even right. Let me know if there is a better way to tell the same story here. Thanks.